web/packages/operations/src/wrap_snow_trac.ts

```diff
@@ -0,0 +1,78 @@
+import { Keyring } from "@polkadot/keyring"
+import { Context, contextConfigFor } from "@snowbridge/api"
+import { cryptoWaitReady } from "@polkadot/util-crypto"
+import { assetRegistryFor } from "@snowbridge/registry"
+import { NeurowebParachain } from "@snowbridge/api/dist/parachains/neuroweb"
+
+const wrapSnowTRAC = async () => {
```

Comment:
Is this wrap/unwrap extrinsic used to switch an asset between native and foreign mode?

It doesn't seem to be used in the API - or do we want to add it to the UI?

Reply:
It will be added to the UI yeah.